cmd/gtxkey/unlock.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/abiosoft/ishell"
+	"github.com/gallactic/gallactic/cmd"
+	"github.com/gallactic/gallactic/keystore"
+)
+
+func Unlock() func(c *ishell.Context) {
+	return func(c *ishell.Context) {
+		// open keystore
```

Comment:
Technically here we get the *instance* of keystore. It's loaded before.

Reply:
resolved  in [commit](https://github.com/gallactic/gallactic/pull/76/commits/91e618e9ee06fb2418c8cdbc25f9e69f530b6c14)